ui/pages/settings/flask/view-snap/view-snap.js

```diff
@@ -176,7 +176,7 @@ function ViewSnap() {
             >
               {t('snapAccess', [snap.manifest.proposedName])}
             </Typography>
-            <Box width={FRACTIONS.TEN_TWELFTHS}>
+            <Box width="full">
```

Comment:
```suggestion
            <Box width={BLOCK_SIZES.FULL}>
```

Don't forget to import `BLOCK_SIZES` with it :)

Reply:
Updated.